sway/input/cursor.c

```diff
@@ -725,7 +725,6 @@ void dispatch_cursor_button(struct sway_cursor *cursor,
 			while (cont->parent) {
 				cont = cont->parent;
 			}
-			seat_set_focus_container(seat, cont);
```

Comment:
Any bad consequences of doing this?

Reply:
I just moved it into `seatop_begin_move_floating` so no